README.md

```diff
@@ -49,20 +49,21 @@ And you are good to go! You can start [exploring our tutorial notebooks](https:/
 
 ### Example
 
-To test this scenario we have available one example of a protein PDB file - [download it here](inductiva/resources/alanine.pdb).
-
 ```python
  import inductiva
 
 inductiva.api_key = "YOUR_API_KEY"
 
+ # Load example protein
+ insulin = inductiva.molecules.examples.load_insulin()
```

Comment:
Change here to `inductiva.examples.load_insulin()`. Based on an exchange that is already account for.

Reply:
You are going to change it?

Reply:
But the PR hasn't been merged and it can change! I will merge like this and when the other one is ready I will alter it quickly.

Reply:
Wait. I am getting late to this discussion :)

I don't think we should have 

```
inductiva.examples.load_insulin()
```

since the examples are not applicable accross domains, as this structure seems to suggest.
I will post an issue on this.

Reply:
@sarmento Look into this PR https://github.com/inductiva/inductiva/pull/600/files